.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
It would be better to add line break because of `insert_final_newline = true`?

Reply:
“I’ll leave this up to you!

Reply:
Then I will handle this in another PR!

Reply:
Thank you, that’s very helpful, too!

Reply:
See #32